puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging_build_deb_coreproject.yaml

```diff
@@ -32,7 +32,7 @@
           values:
           - debian
     execution-strategy:
-      combination-filter: '((onlyarch == "all") || onlyarch == arch) && ((onlyos == "all") || onlyos == os) && !((version != "1.16" || (arch == "armv7" || arch == "armv8")) && os == "trusty") && !((version == "1.15" || version == "1.16") && os == "stretch")'
+      combination-filter: '((onlyarch == "all") || onlyarch == arch) && ((onlyos == "all") || onlyos == os) && !((version != "1.15" || (arch == "armv7" || arch == "armv8")) && os == "trusty") && !((version != "1.15" || version != "1.16") && os == "jessie") && !((version == "1.14" || version == "1.15") && os == "stretch")'
```

Comment:
Nit: the braces are a bit inconsistent. the ```"all"``` arch has them but ```arch``` doesn't. Same for onlyos.

I wonder if we can write this in multi line because for my understanding I rewrote it to:
```
(onlyarch == "all" || onlyarch == arch)
&& (onlyos == "all" || onlyos == os)
&& !(os == "trusty" && (version != "1.15" || (arch == "armv7" || arch == "armv8")))
&& !(os == "jessie" && (version != "1.15" || version != "1.16"))
&& !(os == "stretch" && (version == "1.14" || version == "1.15"))
```

Reply:
I would need to test this, without tryiong I'd say no.